django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -888,6 +895,7 @@ input[type="submit"], button {
     }
 
     .login .submit-row {
+        margin: 0;
```

Comment:
This change is unrelated, please revert.

Reply:
sure